### PR TITLE
change ls to searchparams

### PR DESCRIPTION
**Pull request by YuliyaSosnovskaya**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -131,13 +141,19 @@ function appendUsers(users) {
         const activeUserAvatar = activeUserEl.getElementsByClassName('container-for-avatar')[0];
         activeUserAvatar.classList.remove('container-for-avatar-hover');
         filterUserElAtrId = userEl.id;
-        localStorage.setItem('filter', JSON.stringify({userId: Number(filterUserElAtrId.split('-')[1])}));
+        const userId = filterUserElAtrId.split('-')[1];
+        if (Number(userId)) {
+          setSearchParams('byUser', Number(filterUserElAtrId.split('-')[1]));
+        } else {
+          // implement function to delete 'byUser' (key) from url and use it here
```
Comment (sergen13rs) [inline index.js]: do you still need this comment?

File: index.js [hunk 2]
```diff
@@ -157,11 +173,12 @@ export function appendTaskInColumn() {
   };
 
   let tasksFromLS = JSON.parse(localStorage.getItem('tasks'));
-  let filterFromLS = JSON.parse(localStorage.getItem('filter'));
+  let userId = Number(getSearchParams('byUser'));
+  // let filterFromLS = JSON.parse(localStorage.getItem('filter'));
```
Comment (sergen13rs) [inline index.js]: do you still need this comment?

File: tasks.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import {createModal} from './modal.js';
-import {appendTaskInColumn, counterForDeleted} from './index.js';
+import {appendTaskInColumn, counterForDeleted, getSearchParams, setSearchParams} from './index.js';
```
Comment (sergen13rs) [inline tasks.js]: for what do you import setSearchParams here?

File: tasks.js [hunk 4]
```diff
@@ -101,12 +102,12 @@ export function appendDeleteTaskInColumn() {
     while (deletedColumn.firstChild) {
       deletedColumn.removeChild(deletedColumn.firstChild);
     };
-
+    const filterFromLS = Number(getSearchParams('byUser'));
     const deletedTasksFromLS = JSON.parse(localStorage.getItem('deletedTasks'));
-    let filterFromLS = JSON.parse(localStorage.getItem('filter'));
-    const filteredTasks = filterFromLS.userId === 0
+    // let filterFromLS = JSON.parse(localStorage.getItem('filter'));
```
Comment (sergen13rs) [inline tasks.js]: do you still need this comment?

File: tasks.js [hunk 5]
```diff
@@ -101,12 +102,12 @@ export function appendDeleteTaskInColumn() {
     while (deletedColumn.firstChild) {
       deletedColumn.removeChild(deletedColumn.firstChild);
     };
-
+    const filterFromLS = Number(getSearchParams('byUser'));
```
Comment (sergen13rs) [inline tasks.js]: why filterFromLS? change name of this variable

**Review by sergen13rs — COMMENTED** [review]
fix comments please

File: tasks.js (hunk 3 above)
Comment (YuliyaSosnovskaya) [reply tasks.js:2]: for appendDeleteTaskInColumn in tasks.js

**Review by sergen13rs — APPROVED** [review]
nice. keep going!

**Pull request merged**